business_requirement_deliverable_report/reports/business_requirement_deliverable_report.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
```

Comment:
This will allow user to change or modify a bit of the report without been restored after updating this module.